apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -0,0 +1,694 @@
+        if (apiFunctionLike.tsdocComment) {
+          if (apiFunctionLike.tsdocComment.returnsBlock) {
+            this._appendSection(output, apiFunctionLike.tsdocComment.returnsBlock.content);
+          }
+        }
+      }
+
+    }
+  }
+
+  private _createTitleCell(apiItem: ApiItem): DocTableCell {
+    const configuration: TSDocConfiguration = this._tsdocConfiguration;
+
+    return new DocTableCell({ configuration }, [
+      new DocParagraph({ configuration }, [
+        new DocLinkTag({
+          configuration,
+          tagName: '@link',
+          linkText: Utilities.getConciseSignature(apiItem),
+          urlDestination: this._getLinkFilenameForApiItem(apiItem)
+        })
+      ])
+    ]);
+  }
+
+  private _createDescriptionCell(apiItem: ApiItem): DocTableCell {
+    const configuration: TSDocConfiguration = this._tsdocConfiguration;
+
+    const section: DocSection = new DocSection({ configuration });
+
+    if (ApiReleaseTagMixin.isBaseClassOf(apiItem)) {
+      if (apiItem.releaseTag === ReleaseTag.Beta) {
+        section.appendNodesInParagraph([
+          new DocEmphasisSpan({ configuration, bold: true, italic: true }, [
+            new DocPlainText({ configuration, text: '(BETA)' })
+          ]),
+          new DocPlainText({ configuration, text: ' ' })
+        ]);
+      }
+    }
+
+    if (apiItem instanceof ApiDocumentedItem) {
+      if (apiItem.tsdocComment !== undefined) {
+        this._appendAndMergeSection(section, apiItem.tsdocComment.summarySection);
+      }
+    }
+
+    return new DocTableCell({ configuration }, section.nodes);
+  }
+
+  private _createModifiersCell(apiItem: ApiItem): DocTableCell {
+    const configuration: TSDocConfiguration = this._tsdocConfiguration;
+
+    const section: DocSection = new DocSection({ configuration });
+
+    if (ApiStaticMixin.isBaseClassOf(apiItem)) {
+      if (apiItem.isStatic) {
+        section.appendNodeInParagraph(new DocCodeSpan({ configuration, code: 'static' }));
+      }
+    }
+
+    return new DocTableCell({ configuration }, section.nodes);
+  }
+
+  private _createResultTypeCell(apiItem: ApiItem): DocTableCell {
+    const configuration: TSDocConfiguration = this._tsdocConfiguration;
+
+    const section: DocSection = new DocSection({ configuration });
+
+    if (ApiResultTypeMixin.isBaseClassOf(apiItem)) {
+      section.appendNodeInParagraph(new DocCodeSpan({ configuration, code: apiItem.resultTypeSignature }));
+    }
+
+    return new DocTableCell({ configuration }, section.nodes);
+  }
+
+  private _writeBreadcrumb(output: DocSection, apiItem: ApiItem): void {
+    output.appendNodeInParagraph(new DocLinkTag({
+      configuration: this._tsdocConfiguration,
+      tagName: '@link',
+      linkText: 'Home',
+      urlDestination: './index'
+    }));
+
+    for (const hierarchyItem of apiItem.getHierarchy()) {
+      switch (hierarchyItem.kind) {
+        case ApiItemKind.Model:
+        case ApiItemKind.EntryPoint:
+          break;
```

Comment:
Why are models and entrypoints excluded from the breadcrumb? (This is me just trying to understand the code, really).

Reply:
There is only one model, and it acts as the top-level container for everything.  I had originally considered calling this class `ApiDatabase`, but "model" seemed like a more general term (similar to Model-View-Controller), since it isn't very databasey.  Another name I considered was `ApiPackageSet` or `ApiPackageCollection`, but I liked the folder "src/api/model" better than "src/api/packageSet".  And we also have an `ApiModelGenerator` class that needs to be easy to talk about.

@natalieethell @iclanton let me know if you have any thoughts about that.  The "model" is the second most important part of this API.

---
In reply to: [236369741](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236369741) [](ancestors = 236369741)

Reply:
The entry point is excluded from the breadcrumb because API Extractor does not *yet* support multiple entry points.  If/when we implement that, then we would show it somehow in the breadcrumb.

---
In reply to: [236570281](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236570281) [](ancestors = 236570281,236369741)

Reply:
Got it. "Model" makes sense to me.